paddle/fluid/operators/jit/test.cc

```diff
@@ -640,6 +640,66 @@ void TestNCHW16CMulNCKernel() {
   }
 }
 
+template <paddle::operators::jit::KernelType KT, typename T, typename PlaceType>
+void TestLayerNormKernel() {
+  VLOG(10) << "===== Test JITKernel " << jit::to_string(KT);
+  const int n = 3, x_dim_0 = 50, x_dims_1 = 200;
+  const T epsilon = 9.99999975e-06;
+  auto ref = jit::GetRefer<KT, jit::LayerNormTuples<T>>();
+  EXPECT_TRUE(ref != nullptr);
+  int sz = n * x_dim_0 * x_dims_1;
+  int left = n * x_dim_0, right = x_dims_1;
+  std::vector<T> x(sz), mean(left), var(left), scale(right), bias(right),
+      outref(sz);
+  std::vector<T> outtgt(sz);
+  RandomVec<T>(sz, x.data(), -2.f, 2.f);
+  RandomVec<T>(left, mean.data(), -2.f, 2.f);
+  RandomVec<T>(left, var.data(), -2.f, 2.f);
+  RandomVec<T>(right, scale.data(), -2.f, 2.f);
+  RandomVec<T>(right, bias.data(), -2.f, 2.f);
+
+  const T* scale_data = scale.data();
+  const T* bias_data = bias.data();
+  T* x_data = x.data();
+  T* mean_data = mean.data();
+  T* var_data = var.data();
+  T* outref_data = outref.data();
+  T* outtgt_data = outtgt.data();
+  auto tgt = jit::Get<KT, jit::LayerNormTuples<T>, PlaceType>(0);
```

Comment:
You 'd better use `TestAllImpls`, this function will test all implementations of this kernel type. 

Same as below.

Reply:
Done

Reply:
Done